### ISSUE TITLE
Using Content Library with govc

### ISSUE THREAD

**Issue by gsexton** [vmware/govmomi#1433]
I'm trying to figure out how to deploy an OVA from a content library. I've figured out how to use import.spec to create a spec file to control the import.

vcenter.deploy seems to be the right path. It takes a library item but it doesn't take a spec options file.

import.ova looks promising as well. It supports a json spec file, but I don't see how to specify a library item. Is there some way to specify a library item?

Am I overlooking something? Is there a way to deploy a library item, and specify the options (e.g. Network, DNS, etc)?

I'm afraid I've zero Go language experience, so modifying one of the two looks like a daunting task.

**Comment by dougm** [comment]
The govc content library support is still under development, but expected to be completed soon for the upcoming release.  Current example below from the tests, but the command names/flags are going to change and we still need to add spec options support.

https://github.com/vmware/govmomi/blob/606c1536dd62299ff2bec378e08ecf4dfdc6bf22/object/virtual_device_list.go#L200-L201

**Comment by dougm** [comment]
Support for `-options` is included now that #1433 is merged.  Any feedback would be appreciated, thanks.

**Comment by mikeschinkel** [comment]
For those who find this issue when Googling how to deploy a OVF from the Content Library and are puzzled by the above like I was, here is the working syntax as of current version of `govc`:

```
govc library.deploy /FolderName/name-of-template.ovf new-vm-name
```

You may have to specify some other arguments too given your specific use-case, but govc will display a list of options if you do not provide enough.

Hope this helps.
